src/components/FilterGroup.tsx

```diff
@@ -75,11 +79,17 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const collapsibleLabelCssClasses: CollapsibleLabelCssClasses = useMemo(() => {
+    return {
+      label: cssClasses.titleLabel
+    };
+  }, [cssClasses]);
+
   function renderTitle() {
     return collapsible
-      ? <CollapsibleLabel label={title} />
+      ? <CollapsibleLabel label={title} customCssClasses={collapsibleLabelCssClasses} />
       : (title &&
-        <div className='text-neutral-dark text-sm font-medium text-left mb-4'>
+        <div className={twMerge('mb-4', builtInCollapsibleLabelCssClasses.label, collapsibleLabelCssClasses.label)}>
```

Comment:
I was thinking we could put this in the useMemo like `label: twMerge('mb-4', builtInCollapsibleLabelCssClasses.label, collapsibleLabelCssClasses.label)` and then have this be `className={collapsiblelabelCssClasses.label}` but it's ok

Reply:
`builtInCollapsibleLabelCssClasses` comes from `CollapsibleLabel` so I don't want to merge it back into `collapsibleLabelCssClasses` because then it'll merge in twice